--- /mbed-ST_LSM6DSOX/lsm6dsox_reg.c	(last commit)
+++ /github-stm32duino_LSM6DSOX/lsm6dsox_reg.c (commit d00c52d7eebe36cc1abd318d934f48852e3b00ea)
@@ -83,59 +83,59 @@
   * @{
   *
   */
-float_t lsm6dsox_from_fs2_to_mg(int16_t lsb)
-{
-  return ((float_t)lsb) * 0.061f;
-}
-
-float_t lsm6dsox_from_fs4_to_mg(int16_t lsb)
-{
-  return ((float_t)lsb) * 0.122f;
-}
-
-float_t lsm6dsox_from_fs8_to_mg(int16_t lsb)
-{
-  return ((float_t)lsb) * 0.244f;
-}
-
-float_t lsm6dsox_from_fs16_to_mg(int16_t lsb)
-{
-  return ((float_t)lsb) *0.488f;
-}
-
-float_t lsm6dsox_from_fs125_to_mdps(int16_t lsb)
-{
-  return ((float_t)lsb) *4.375f;
-}
-
-float_t lsm6dsox_from_fs500_to_mdps(int16_t lsb)
-{
-  return ((float_t)lsb) *17.50f;
-}
-
-float_t lsm6dsox_from_fs250_to_mdps(int16_t lsb)
-{
-  return ((float_t)lsb) *8.750f;
-}
-
-float_t lsm6dsox_from_fs1000_to_mdps(int16_t lsb)
-{
-  return ((float_t)lsb) *35.0f;
-}
-
-float_t lsm6dsox_from_fs2000_to_mdps(int16_t lsb)
-{
-  return ((float_t)lsb) *70.0f;
-}
-
-float_t lsm6dsox_from_lsb_to_celsius(int16_t lsb)
-{
-  return (((float_t)lsb / 256.0f) + 25.0f);
-}
-
-float_t lsm6dsox_from_lsb_to_nsec(int16_t lsb)
-{
-  return ((float_t)lsb * 25000.0f);
+float lsm6dsox_from_fs2_to_mg(int16_t lsb)
+{
+  return ((float)lsb) * 0.061f;
+}
+
+float lsm6dsox_from_fs4_to_mg(int16_t lsb)
+{
+  return ((float)lsb) * 0.122f;
+}
+
+float lsm6dsox_from_fs8_to_mg(int16_t lsb)
+{
+  return ((float)lsb) * 0.244f;
+}
+
+float lsm6dsox_from_fs16_to_mg(int16_t lsb)
+{
+  return ((float)lsb) *0.488f;
+}
+
+float lsm6dsox_from_fs125_to_mdps(int16_t lsb)
+{
+  return ((float)lsb) *4.375f;
+}
+
+float lsm6dsox_from_fs500_to_mdps(int16_t lsb)
+{
+  return ((float)lsb) *17.50f;
+}
+
+float lsm6dsox_from_fs250_to_mdps(int16_t lsb)
+{
+  return ((float)lsb) *8.750f;
+}
+
+float lsm6dsox_from_fs1000_to_mdps(int16_t lsb)
+{
+  return ((float)lsb) *35.0f;
+}
+
+float lsm6dsox_from_fs2000_to_mdps(int16_t lsb)
+{
+  return ((float)lsb) *70.0f;
+}
+
+float lsm6dsox_from_lsb_to_celsius(int16_t lsb)
+{
+  return (((float)lsb / 256.0f) + 25.0f);
+}
+
+float lsm6dsox_from_lsb_to_nsec(int16_t lsb)
+{
+  return ((float)lsb * 25000.0f);
 }
 
 /**
